src/lesson10/Main.java

```diff
@@ -0,0 +1,70 @@
+package lesson10;
```

Comment:
Every class must be in separate file.
On the top of each class add description from HW description